featuretools/entityset/entityset.py

```diff
@@ -268,84 +267,67 @@ def add_relationship(self, relationship):
         self.reset_data_description()
         return self
 
-    def get_pandas_data_slice(self, filter_entity_ids, index_eid,
+    def get_pandas_data_slice(self, filter_eid, index_eid,
                               instances, entity_columns=None,
                               time_last=None, training_window=None,
                               verbose=False):
```

Comment:
I think we can get rid of `verbose` in `calculate_all_features` too? It is used in the function, but the only caller does not pass it so it will always be `False`.